Packs/Illumio/Playbooks/playbook-Rapid_ransomware_containment_-_Illumio.yml

```diff
@@ -0,0 +1,1085 @@
+      virtual_service:
+        complex:
+          root: Illumio.VirtualService
+          accessor: href
+      workloads:
+        complex:
+          root: Illumio.TrafficAnalysis.Workload
+          accessor: href
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": -510,
+          "y": 1545
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  '5':
+    id: '5'
+    taskid: 8850d503-049f-44d1-8e12-49aefe9f2878
+    type: regular
+    task:
+      id: 8850d503-049f-44d1-8e12-49aefe9f2878
+      version: -1
+      name: Create ruleset
+      description: Creates a ruleset with a unique name. Until provisioned with the 'illumio-object-provision' command, this object will remain in a draft state.
+      script: '|||illumio-ruleset-create'
+      type: regular
+      iscommand: true
+      brand: ''
+    nexttasks:
+      '#none#':
+      - '7'
+    scriptarguments:
+      name:
+        complex:
+          root: inputs.Protocol
+          transformers:
+          - operator: trim
+          - operator: SetIfEmpty
+            args:
+              applyIfEmpty: {}
+              defaultValue:
+                value:
+                  simple: TCP
+          - operator: concat
+            args:
+              prefix:
+                value:
+                  simple: '-'
+              suffix: {}
+          - operator: concat
+            args:
+              prefix:
+                value:
+                  simple: inputs.Port
+                iscontext: true
+              suffix: {}
+          - operator: concat
+            args:
+              prefix:
+                value:
+                  simple: RS-XSOAR-
+              suffix: {}
+          - operator: replace
+            args:
+              limit: {}
+              replaceWith: {}
+              toReplace:
+                value:
+                  simple: ' '
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": -510,
+          "y": 1750
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  '7':
+    id: '7'
+    taskid: d5ad8e4e-b93a-4bba-8f25-bd4ff851f173
+    type: regular
+    task:
+      id: d5ad8e4e-b93a-4bba-8f25-bd4ff851f173
+      version: -1
+      name: Create rule
+      description: Creates & assigns rules to a particular ruleset. Added or updated Rules will remain in draft state until their containing Rule Set is provisioned using the "illumio-object-provision" command.
+      script: '|||illumio-rule-create'
+      type: regular
+      iscommand: true
+      brand: ''
+    nexttasks:
+      '#none#':
+      - '12'
+    scriptarguments:
+      consumers:
+        simple: /orgs/1/sec_policy/draft/ip_lists/1
```

Comment:
Per our discussion, let's change these hardcoded HREFs as the org ID may be different and the IP list ID may also be different.

Reply:
Yes will directly call the IP list href